transcribed/page1160.txt

```diff
@@ -0,0 +1,23 @@
+001             ;*SYSTEM* JOBS                                                  PAGE 116
+002
+003     117 016 SCORE:   SKIPL SUPCOR
+004     017 002          PUSHJ P,UPL3
+005     116 017          PUSHJ P,SCORE4
+006     117 013          CLEARM @SYSL
+007     117 013          HRLX A,SYSL
```

Comment:
HRLZ

Reply:
Do we know what instructions can exist here? I agree with you that it looks more probable to be HRLZ than HRLX though. I'm just thinking that a complete list of instructions could be helpful to spot typos.

Reply:
Yes, we know all the standard pre-defined instruction mnemonics.  HRLZ is one of them: "half-word right to left, zero other half".  HRLX is not a "valid" instruction, although of course any program can provide a definition.

This is a very compact description of the PDP-10 (and mostly -6) instruction set: https://www.ultimate.com/phil/pdp10/instructionset.jpg
Perhaps too compact!  But you can see the half-word instruction family to the upper left.